includes/class-wc-stripe-webhook-handler.php

```diff
@@ -1269,6 +1335,17 @@ public function process_deferred_webhook( $webhook_type, $additional_data ) {
 					throw new Exception( "Unsupported webhook type: {$webhook_type}" );
 					break;
 			}
+			/**
+			 * Fires after a webhook has been processed, but before we respond to Stripe.
+			 * This allows for custom processing of the webhook after it has been processed.
+			 *
+			 * @since 9.8.0
+			 *
+			 * @param string $webhook_type The type of webhook that was processed.
+			 * @param object $notification The webhook data sent from Stripe.
+			 * @param WC_Order $order The order being processed by the webhook.
+			 */
+			do_action( 'wc_stripe_webhook_processed', (string) $webhook_type, $notification, $this->resolved_order );
 		} catch ( Exception $e ) {
```

Comment:
We should catch `Throwable` to be more robust against bad code. 😬  
```suggestion
		} catch ( Throwable $e ) {
```

Reply:
Updated in 6f832701806f6dfa2f640a6b354063414def16f8.

Reply:
And centralised in 479c7b2c4bf58d5a92efe9b0e31a77e836191226.